.gitignore

```diff
@@ -59,3 +59,6 @@ typings/
 
 # MacOS system files
 .DS_Store
+
+# JetBrains IDE
+.idea
```

Comment:
I made this change, but it is showing up in your pull request because you didn't merge the latest master into your branch before committing your code. This happens sometimes. You can fix it by running the following commands in the terminal:

```bash
$ git checkout master
$ git pull upstream master
$ git push origin HEAD
$ git checkout rest-draft
$ git rebase master
$ git push -f
```

This process is called "rebasing" and should be done sparingly. Basically, it rewrites git history so that all of the commits you made come after the most recent master branch is merged in. It first undoes all of your commits, then merges in the latest changes from master to your local branch, then replays each of your commits. Finally, `git push -f` forces your changes to be pushed to your GitHub repository, rewriting the git history there.